public/quotes.js

```diff
@@ -154,5 +154,9 @@ var quotes = [
   {
     quote: "We have developers who kind of develop",
     author: "Andreas"
+  },
+  {
+    quote: "...(typing)...",
+    author: "Andreas"
   }
```

Comment:
@fespinoza: This is not a JSON file. :wink: 

@onmyway133: Agree, that would probably be nice, but we don't have support for formatting the quotes now. Would need to add support for something like Markdown or perhaps even just write it in HTML in here for that.